### PR TITLE
Update docs for merging S&T and L&L

### PR DESCRIPTION
**Pull request by jonallured**
After some chatting on Slack:

https://artsy.slack.com/archives/C9ZUJE65V/p1557411649003100

We're looking to use the L&L time slot for both events and alternate weeks.

h/t @alloy for the nudge here!! ❤️

### CODE REVIEW

File: events/lunch-and-learn.md [hunk 1]
```diff
@@ -1,25 +1,36 @@
...
+[alan]: https://twitter.com/alanjay1/
+[alan_post]: https://artsy.github.io/blog/2016/09/22/professional-development-at-artsy-engineering/
+
+> Every Thursday, we do a Lunch-and-Learn session. Historically, we mostly
+> showed off tech we use internally. But over about a year, most of our tech
+> stack has been presented this way, so we also bring in engineers we know at
 > other companies to share what they’re working on.
 
+On the off weeks we have a [Show & Tell](show-and-tell.md) and those weeks where
+we can't find a Lunch & Learn speaker we fall back to doing Show & Tell.
```
Comment (anandaroop) [inline events/lunch-and-learn.md]: 👍

File: events/show-and-tell.md [hunk 2]
```diff
@@ -5,27 +5,24 @@ description: Internal presentations with no preparation
...
-an open mic-style, developers step forward to show and tell!
+In order to better know what our teammates are up to, every other week we set
+aside an hour and in an open mic-style setting show each other what we're
+working on. These short talks/demos/code reviews are not meant to be polished
+presentations, but instead impromptu ways we can share lessons learned and cool
+things discovered!
+
+This happens at the same time slot as [Lunch & Learn](lunch-and-learn.md) and we
+alternate between the two events. On Lunch & Learn weeks without a speaker lined
+up, we fall back to doing a bonus Show & Tell.
```
Comment (anandaroop) [inline events/show-and-tell.md]: Nice!

This diff mostly looks like line-length noise, but the bits here and above LGTM 👍

File: events/lunch-and-learn.md [hunk 3]
```diff
@@ -56,57 +67,67 @@ Here are some examples of internal presentations:
...
+the presentation mainly being a series of demos, and showing code architecture.
 
 ## Schedule
 
-- Arrive at Artsy for 12:15pm (we're [401 Broadway, 26th Floor, 10013][401] - Canal St is the closest station)
-- Our receptionist on 26 will get in touch with whoever you've been in contact with, probably [Orta][] or [Ash][]
+- Arrive at Artsy by 12:15pm ([401 Broadway, 26th Floor, 10013][401] - Canal St
+  is the closest station)
+- Our receptionist on 26 will get in touch with whoever you've been in contact
+  with, probably [Orta][] or [Ash][]
```
Comment (ashfurrow) [inline events/lunch-and-learn.md]:
```suggestion
  with, probably [Ash][]
```

Open to someone else stepping up as a second L&L NYC point-person. @anandaroop would you be interested?

Comment (anandaroop) [reply events/lunch-and-learn.md]: Yeah let’s do it 💪🏽

File: events/lunch-and-learn.md [hunk 4]
```diff
@@ -56,57 +67,67 @@ Here are some examples of internal presentations:
...
+We'd like to get a copy of your slides, too. Of course, we're happy to send you
+a copy of the recording.
 
-**If you don't want to be recorded, no problem**. We understand that you might talk and show things that are not
-for public consumption. We will not share anything externally without your express permission.
+**If you don't want to be recorded, no problem**. We understand that you might
+talk and show things that are not for public consumption. We will not share
+anything externally without your express permission.
 
 [orta]: https://github.com/orta
```
Comment (ashfurrow) [inline events/lunch-and-learn.md]:
```suggestion
[Roop]: https://github.com/anandaroop
```

**Pull request merged**